src/content/resources/kotlin/native/creating-c-bindings.md

```diff
@@ -0,0 +1,113 @@
+---
+authors:
+  - "supergrecko#3434"
+created_at: "2019/08/23"
+title: "Creating C bindings with Kotlin/Native"
+external_resources:
+  - "https://kotlinlang.org/docs/reference/native/c_interop.html"
+  - "https://github.com/JetBrains/kotlin-native/releases"
+---
+
+# Interoperability with C
```

Comment:
Start the headings at `##` for styles to be properly set. The h1 (`#`) will use `title` front the frontmatter.